src/Microsoft.DotNet.ProjectModel/Compilation/LibraryExporter.cs

```diff
@@ -184,22 +190,24 @@ private LibraryExport ExportProject(ProjectDescription project)
                     Path.GetFullPath(Path.Combine(project.Project.ProjectDirectory, assemblyPath)));
 
                 compileAssemblies.Add(compileAsset);
-                runtimeAssets.Add(pdbPath);
+                runtimeAssets.Add(new LibraryAsset(pdbPath, Path.GetFileName(pdbPath), pdbPath));
             }
             else if (project.Project.Files.SourceFiles.Any())
             {
-                var outputCalculator = project.GetOutputPathCalculator();
-                var assemblyPath = outputCalculator.GetAssemblyPath(_configuration);
+                var outputCalculator = project.GetOutputPathCalculator(_buildBasePath, _rootPath, _configuration, _runtime);
+                var files = outputCalculator.CompilationFiles;
```

Comment:
Is this supposed to mean CompiledFiles? Also, same comment as above about naming and thinking these should belong in their own Object. Maybe the calculator is turning into a Factory of OutputPaths.